### PR TITLE
Use react-scripts

### PR DESCRIPTION
**Pull request by vincemtnz**
So `create-react-app` is rapidly evolving and keeps better track of what's happening in the react ecosystem. With zero configuration you get hot module reloading, and a pretty good build pipeline + tests configured to run with Jest. We won't need to worry about updating 20 dependencies for testing, building, linting, transpiling... etc, and instead we simply need to update the `react-scripts` package ✌️ 

tl;dr: Let someone else maintain our build pipeline 🌻

### CODE REVIEW

**Comment by vincemtnz** [comment]
A few things change:
- Use relative paths for all imports (since we can't customize the webpack config, it's a good thing anyway).
- No sass/less preprocessor for styles [yet](https://github.com/facebookincubator/create-react-app/issues/78), but components + bem style class names might be enough (flat is better than nested). 
- Instead of axios, I've gone for `fetch` since a [polyfill is provided](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/config/polyfills.js#L21) out of the box. There are a few differences introduced by using `fetch` but overall I think it's a better option.
- We now use ENV variables for the build. All ENV variables that we want to use in our react app need to be named `REACT_APP_*`. For example, I've renamed `CHAMPAIGN_URL` to `REACT_APP_CHAMPAIGN_URL`. We will need to set the relevant environment variables in CI (and in the `.env` file for development. See `.env.example`).

**Comment by osahyoun** [comment]
💯
